tests/test_redact.py

```diff
@@ -22,12 +23,18 @@ def svs_input_path(data_dir, request) -> Path:
     return data_dir / "input" / request.param
 
 
+@freeze_time("2023-05-12 12:12:53")
+def test_create_redact_dir(tmp_path):
+    output_dir = redact.create_redact_dir(tmp_path)
+    assert output_dir.exists()
+
+
+@freeze_time("2023-05-12 12:12:53")
 def test_redact_svs(svs_input_path, tmp_path, override_rule_set):
     redact.redact_images(svs_input_path, tmp_path, override_rule_set)
 
-    svs_output_file = tmp_path / "REDACTED_test_svs_image_blank.svs"
-    assert svs_output_file.exists()
-    svs_output_file_bytes = svs_output_file.read_bytes()
+    output_file = tmp_path / "Redacted_2023-05-12_12-12-53/REDACTED_test_svs_image_blank.svs"
```

Comment:
We cannot form paths this way on Windows. Each component following the `Path` must be its own string.

Reply:
fixed in a0c2bf04704b088fb3ba1cfdeb17ff930b8983bf